rust/worker/chroma_config.yaml

```diff
@@ -36,7 +36,7 @@ query_service:
                     max_concurrent_requests: 15
     log:
         grpc:
-            host: "logservice.chroma"
+            host: "rust-log-service.chroma"
```

Comment:
we need to make sure this won't break deployed envs, i forget if they pick up defaults from here